meson_options.txt

```diff
@@ -33,3 +33,9 @@ option('systemd',
   value : true,
   description : 'Build systemd unit files'
 )
+
+option('elogind',
```

Comment:
I think a better option here would be something like `logind-provider=systemd|elogind`. More expressive, especially since this is an either/or, so it should be a meson `combo` option.